openapi/paths/storefront/invoices@{id}@send-email.yaml

```
@@ -0,0 +1,27 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Storefront
+  tags:
+    - Storefront invoices
+  summary: Resend an Invoice email
```

nitpick: capitalization 
```suggestion
  summary: Resend an invoice email
```